Rakefile

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-ruby_19 = RUBY_VERSION > '1.9'
+ruby_one_nine = RUBY_VERSION > '1.9'
```

Comment:
In this case, we can know that this will always be true.

The gemspec file now requires the Ruby version to be 2.3+.

So, we could seize the moment and get rid of this variable, since it will always be truthy.

Would you like to do that?

Reply:
Yup! Just a second.

Reply:
Thanks a lot!